### PR TITLE
More cleanup

### PR DESCRIPTION
**Pull request by max-schaefer**
Mostly related to the WebSocket model introduced in #107 and refined in #109.

Renaming the QLL file from `Websocket.qll` to `WebSocket.qll` is technically a breaking change, but we only just introduced it, so I think it's fine. (And also one wouldn't normally import that module directly anyway.)

### CODE REVIEW

File: ql/src/go.qll [hunk 1]
```diff
@@ -34,6 +34,6 @@ import semmle.go.frameworks.SQL
 import semmle.go.frameworks.XPath
```
Comment (sauyon) [inline ql/src/go.qll:34]: Should we realphabetize this in this PR?

File: ql/test/experimental/frameworks/Gin/vendor/github.com/gin-gonic/gin/stub.go [hunk 2]
```diff
@@ -259,7 +259,7 @@ func (_ *Context) IsAborted() bool {
 	return false
 }
 
-func (_ *Context) IsWebsocket() bool {
+func (_ *Context) IsWebSocket() bool {
```
Comment (sauyon) [inline ql/test/experimental/frameworks/Gin/vendor/github.com/gin-gonic/gin/stub.go:262]:
```suggestion
func (_ *Context) IsWebsocket() bool {
```

I assume this change was unintended?

File: ql/test/library-tests/semmle/go/frameworks/WebSocket/vendor/github.com/sacOO7/gowebsocket/stub.go [hunk 3]
```diff
@@ -29,7 +29,7 @@ func New(_ string) Socket {
 
 type Socket struct {
 	Conn              interface{}
-	WebsocketDialer   interface{}
+	WebSocketDialer   interface{}
```
Comment (sauyon) [inline ql/test/library-tests/semmle/go/frameworks/WebSocket/vendor/github.com/sacOO7/gowebsocket/stub.go:32]:
```suggestion
	WebsocketDialer   interface{}
```

ditto

File: ql/test/query-tests/Security/CWE-918/vendor/github.com/sacOO7/gowebsocket/stub.go [hunk 4]
```diff
@@ -29,7 +29,7 @@ func New(_ string) Socket {
 
 type Socket struct {
 	Conn              interface{}
-	WebsocketDialer   interface{}
+	WebSocketDialer   interface{}
```
Comment (sauyon) [inline ql/test/query-tests/Security/CWE-918/vendor/github.com/sacOO7/gowebsocket/stub.go:32]:
```suggestion
	WebsocketDialer   interface{}
```

File: ql/src/semmle/go/Packages.qll [hunk 5]
```diff
@@ -26,10 +26,13 @@ class Package extends @package {
...
- * Gets the Go import string that may identify a package in module `mod` with the given path,
- * possibly modulo semantic import versioning.
+ * Gets an import path that identifies a package in module `mod` with the given path,
+ * possibly modulo [semantic import versioning](https://github.com/golang/go/wiki/Modules#semantic-import-versioning).
+ *
+ * For example, `package("github.com/go-pg/pg", "types")` gets an import path that can
+ * refer to `"github.com/go-pg/pg/types"`, but also to `"github.com/go-pg/pg/v10/types"`.
  */
 bindingset[result, mod, path]
 string package(string mod, string path) {
```
Comment (sauyon) [inline ql/src/semmle/go/Packages.qll:36]: A thought for the future: should we add a `version` parameter to match the version as well?

File: ql/src/go.qll (hunk 1 above)
Comment (max-schaefer) [reply ql/src/go.qll:34]: Good plan.

File: ql/test/experimental/frameworks/Gin/vendor/github.com/gin-gonic/gin/stub.go (hunk 2 above)
Comment (max-schaefer) [reply ql/test/experimental/frameworks/Gin/vendor/github.com/gin-gonic/gin/stub.go:262]: Whoops, yes; let me revert those changes.

File: ql/src/semmle/go/Packages.qll (hunk 5 above)
Comment (max-schaefer) [reply ql/src/semmle/go/Packages.qll:36]: Could do, in the future.

**Comment by max-schaefer** [comment]
Thanks, @sauyon, comments addressed.

**Comment by max-schaefer** [comment]
Added one more (unrelated) fix.

**Pull request merged**